mLRS/CommonTx/esp.h

```diff
@@ -227,12 +228,12 @@ void tTxEspWifiBridge::passthrough_do_flashing(void)
     delay_ms(100); // 10 ms is too short, ESP8285 needs more time
     esp_gpio0_high();
     delay_ms(10);
+    com->SetBaudRate(baudrate); // Standard tools should specify 115200 to avoid baudrate change
 #endif
 
+    ser->SetBaudRate(baudrate);
```

Comment:
why does it have to come before leds.Initxxx() ?

Reply:
I just wanted it near the previous line so it would be more clear we are setting the speed of both serial ports.